includes/emails/class-wcs-email-customer-notification.php

```diff
@@ -267,30 +267,43 @@ public function get_default_additional_content() {
 	 * @return bool
 	 */
 	public function should_send_reminder_email( $subscription ) {
-		$should_skip = [];
-
 		if ( ! $this->is_enabled() ) {
-			$should_skip[] = __( 'Reminder emails disabled.', 'woocommerce-subscriptions' );
-		} else {
-			if ( ! WC_Subscriptions_Email_Notifications::should_send_notification() ) {
-				$should_skip[] = __( 'Not a production site', 'woocommerce-subscriptions' );
-			}
+			return $this->log_reminder_email_not_sent( $subscription, __( 'Reminder emails disabled.', 'woocommerce-subscriptions' ) );
+		}
 
-			if ( ! $this->get_recipient() ) {
-				$should_skip[] = __( 'Recipient not found', 'woocommerce-subscriptions' );
-			}
+		$skipped_reasons = [];
 
-			if ( WCS_Action_Scheduler_Customer_Notifications::is_subscription_period_too_short( $subscription ) ) {
-				$should_skip[] = __( 'Subscription billing cycle too short', 'woocommerce-subscriptions' );
-			}
+		if ( ! WC_Subscriptions_Email_Notifications::should_send_notification() ) {
+			$skipped_reasons[] = __( 'Not a production site', 'woocommerce-subscriptions' );
```

Comment:
Not essential we change this (because even if I'm right, it's not really new as of this PR), but would the following be more accurate?

```suggestion
			$skipped_reasons[] = __( 'Not a production site, or notifications have been globally disabled', 'woocommerce-subscriptions' );
```

I'm just being mindful here that `::should_send_notification()` in turn calls `::notifications_globally_enabled()`, so the current environment isn't the only reason for this (if I'm reading correctly, at least).

Reply:
I agree. Thanks!